app/components/autocomplete/at_mention_item/at_mention_item.tsx

```diff
@@ -121,28 +122,37 @@ const AtMentionItem = ({firstName = '', isBot, isCurrentUser, isGuest, isShared,
                         show={isGuest}
                         theme={theme}
                     />
-                    {Boolean(name.length) &&
                     <Text
-                        style={style.rowFullname}
                         numberOfLines={1}
-                        testID='at_mention_item.name'
                     >
-                        {name}
-                    </Text>
-                    }
-                    <Text
-                        style={style.rowUsername}
-                        numberOfLines={1}
-                        testID='at_mention_item.username'
-                    >
-                        {isCurrentUser &&
-                        <FormattedText
-                            id='suggestion.mention.you'
-                            defaultMessage=' (you)'
-                        />}
-                        {` @${username}`}
+                        {Boolean(name.length) && (
+                            <Text
+                                style={style.rowFullname}
+                                testID='at_mention_item.name'
+                            >
+                                {name}
+                            </Text>
+                        )}
+                        <Text
+                            style={style.rowUsername}
+                            testID='at_mention_item.username'
+                        >
+                            {isCurrentUser && (
+                                <FormattedText
+                                    id='suggestion.mention.you'
+                                    defaultMessage='(you)'
+                                />
+                            )}
+                            {` @${username}`}
+                        </Text>
                     </Text>
                 </View>
+                {isCustomStatusEnabled && !isBot && (
+                    <CustomStatusEmoji
+                        userID={userId}
+                        style={{marginLeft: 4}}
```

Comment:
1. Can we move this out from inline?
2. Add the same style to Shared channel's `ChannelIcon` component?